sync/src/main/java/tech/pegasys/teku/sync/SyncServiceFactory.java

```diff
@@ -62,6 +64,7 @@ private SyncServiceFactory(
       final BlockImporter blockImporter,
       final PendingPool<SignedBeaconBlock> pendingBlocks,
       final int getStartupTargetPeerCount,
+      final SignatureVerificationService signatureVerificationService,
```

Comment:
Does this need to be declared as `SignatureVerificationService` or could it just be `AsyncBLSSignatureVerifier`?